docs/release_process.rst

```diff
@@ -0,0 +1,165 @@
+
+   Create release builder on Buildbot.
+
+A release can only be created from a *release_staging_<version number>* branch.
+The create release builder interface interrogates `cellml/libcellml <https://github.com/cellml/libcellml>`__ for potential release branches.
+Select the release preparation branch (there should only ever be one), to create the release from, and start the build with the *Start Build* button, :numref:`libcellml_release_process_create_release_builder_interface`.
+
+.. figure:: ./images/release_process/create_release_builder_interface.png
+   :align: center
+   :alt: Buildbot create release interface.
+   :name: libcellml_release_process_create_release_builder_interface
+
+   Create release interface showing the branch a release can be created from.
+
+The create release build creates a release on GitHub and adds tags to identify where the release was created from.
+The current status for the *release_staging_<version number>* branch will be applied to the *release* branch.
+The tagged *release* branch is where the GitHub release will be created from.
+When the release is created GitHub actions will take over to build all the assets.
+The assets here are the binaries (installers, archives) from: Windows; macOS; Ubuntu; the Python wheels; and, the Javascript package.
+
+.. note::
+
+  **Do not** start the finalise release builder until the GitHub actions have finished.
+  This normally takes between 11 - 15 minutes.
+
+Step 4 - Finalising the Release
+===============================
+
+The release is finalised using the *Finalise Release Builder* (:numref:`libcellml_release_process_finalise_release`).
+
+.. figure:: ./images/release_process/finalise_release_builder.png
+   :align: center
+   :alt: Buildbot finalise release builder interface.
+   :name: libcellml_release_process_finalise_release
+
+   Finalise release builder on Buildbot.
+
+The finalise release builder interface interrogates `cellml/libcellml <https://github.com/cellml/libcellml>`__ for release branches, and `end-to-end testing framework <https://github.com/libcellml/end-to-end-testing>`__, and `end-to-end testing database <https://github.com/libcellml/end-to-end-test-database>`__ for tags.
+Choose the values for finalising the release in the finalise release interface, :numref:`libcellml_release_process_finalise_release_builder_interface`.
+
+.. figure:: ./images/release_process/finalise_release_builder_interface.png
+   :align: center
+   :alt: Buildbot finalise release interface.
+   :name: libcellml_release_process_finalise_release_builder_interface
+
+   Finalise release interface.
+
+The finalise release builder will update the libCellML staging website with the API documentation and the developer documentation.
+The user documentation will only be udpated for the release if the end-to-end testing passes.
+To this end, it is important to choose the appropriate end-to-end testing tag and end-to-end testing database tag.
```

Comment:
It's not clear to me what those end-to-end testing and end-to-end testing database tags are about...?